orca-core/src/main/java/com/netflix/spinnaker/orca/clouddriver/utils/CloudProviderAware.java

```diff
@@ -58,9 +58,12 @@ default String getCredentials(Map<String, Object> context) {
 
   // may return a list with 0, 1 or more regions (no guarantees on the ordering)
   default List<String> getRegions(Map<String, Object> context) {
-    String region = (String) context.getOrDefault("region", null);
+    Object region = context.getOrDefault("region", null);
```

Comment:
this sometimes is a string and sometimes a list. I tried to make it handle both. but also wrapped in `try/catch` in the  other files